### ISSUE TITLE
Installation error on Postgres 13 .1

### ISSUE THREAD

**Issue by ArturFormella** [theirix/parray_gin#7]
Hello! 
I have the following error.
```
➜  ~ USE_PGXS=1 sudo pgxn install parray_gin
INFO: best version: parray_gin 1.3.1
INFO: saving /tmp/tmphctdm60v/parray_gin-1.3.1.zip
INFO: unpacking: /tmp/tmphctdm60v/parray_gin-1.3.1.zip
INFO: building extension
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -I. -I./ -I/usr/include/postgresql/13/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2   -c -o src/parray_gin.o src/parray_gin.c
src/parray_gin.c:24:10: fatal error: access/tuptoaster.h: No such file or directory
   24 | #include "access/tuptoaster.h"
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [<builtin>: src/parray_gin.o] Error 1
ERROR: command returned 2: make PG_CONFIG=/usr/bin/pg_config all

```
PostgreSQL 13.1 (Ubuntu 13.1-1.pgdg20.04+1)
Ubuntu 20.04.1 LTS

What can I do?

**Comment by theirix** [comment]
Hello,

Just released a new version 1.3.2 which fixes this bug on Postgres 13.

Please try again, it should be on PGXN now.

**Comment by ArturFormella** [comment]
Hi!
Now works great. Thank you!
